packages/converters/package.json

```diff
@@ -4,9 +4,7 @@
   "private": false,
   "license": "Apache-2.0",
   "type": "module",
-  "files": [
-    "dist/**"
-  ],
+  "files": ["dist/**"],
```

Comment:
I support this change :).

Reply:
Heh this one is always a little rage inducing because syncpack and biome currently have different formatting opinions 😆